src/android/Print.java

```diff
@@ -725,13 +725,20 @@ private int getPrinterSeries(String target) {
 	private void sendToPrintConnect(JSONArray args, final CallbackContext callbackContext) throws JSONException {
 		final String zplString = args.getString(0);
 		byte[] passthroughBytes = null;
+		boolean requiresForeground = false;
 
 		try {
 			passthroughBytes = zplString.getBytes("UTF-8");
 		} catch (UnsupportedEncodingException e) {
 			// Handle exception
 		}
 
+		PackageInfo info = this.mContext.getPackageManager().getPackageInfo("com.zebra.printconnect", 0);
+		long versionCode = info.getLongVersionCode();
```

Comment:
Is it guaranteed for this package info to exist? If not, we should check that it is populated before calling `getLongVersionCode()`

Reply:
It will throw an exception if not found -- included handling for that now.